CI/Dockerfile

```diff
@@ -1,9 +1,36 @@
+# Svalinn DAGMC Dockerfile
+
+# build from repository root directory with command
+# docker build -t dagmc -f CI/Dockerfile .
+
+# Global ARGS set before the first build stage are accessable by all build stages
+ARG EMBREE_BRANCH='v3.6.1'
+ARG geant4_version=10.5.1
```

Comment:
I think we merged an update for up to v10.7.4 (#803)

We'll probably need a new SHA below... which begs the question about whether we should make this a build-arg or not? (Or maybe we don't use the SHA anyway?)

Reply:
change to 10.7.4 which appears to be the newest supported version